lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +237,23 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        link = article_bs.find('a', class_='listing-preview__content')
+        return str(link.get('href'))
```

Comment:
if get returns None, what will be returned?

Reply:
fixed

Reply:
how?

Reply:
fixed?